src/main/resources/liquibase/202209121018-update-primary-key-of-account-config.xml

```diff
@@ -0,0 +1,28 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+
+    <changeSet id="202209121018-1" author="kflahert">
```

Comment:
Unit tests fail because this query won't work on hsqldb.

In order to skip this on hsqldb:

```suggestion
    <changeSet id="202209121018-1" author="kflahert" dbms="postgresql">
```